package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "nodejs_mentoring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "tsc-node": "tsc && nodemon Task2/usersAPI.ts",
```

Comment:
Do you need this line? Looks like you are using `ts-node-dev` and don't need `nodemon` package at all.